docs/showcases/showcases.ts

```diff
@@ -2,14 +2,23 @@ type Showcase = {
     label: string
     src: string
     url: string
+    icon: string // https://devicon.dev/
 }
 
-type Grid<T> = T
-    | [T, T]
-    | [T, T, T]
-    | [T, T, T, T]
+type Grid<T> = T | [T, T] | [T, T, T] | [T, T, T, T]
 
 export default [
-    { label: "Placeholder (this is an ags v1 screenshot)", src: "/astal/showcase/aylur1.png", url: "https://github.com/Aylur/dotfiles" },
-    // add more showcases here
+    {
+        label: "Placeholder (this is an ags v1 screenshot)",
+        src: "/astal/showcase/aylur.png",
+        url: "https://github.com/Aylur/dotfiles",
+        icon: "devicon-javascript-plain",
+    },
+    {
+        label: "Idk I just love oxocarbon",
+        src: "/astal/showcase/contrib1.webp",
```

Comment:
please name the file something else, for example your nickname. anything thats not this generic